runtime/bcverify/staticverify.c

```diff
@@ -1767,6 +1767,19 @@ j9bcv_verifyClassStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile,
 			goto _formatError;
 		}
 
+		/* The requirement for taking no arguments was introduced in Java SE 9.
```

Comment:
Does this only apply to Java 9?  It'll likely need a `if (J2SE_VERSION(vm) >= J2SE_19) ` check